spec/models/ticket_purchase_spec.rb

```diff
@@ -2,118 +2,136 @@
 
 describe TicketPurchase do
 
-  describe 'validations' do
-    it 'has a valid factory' do
-      expect(build(:ticket_purchase)).to be_valid
+  # describe 'validations' do
```

Comment:
Sorry my bad I was testing something here so I commented all other specs and forgot to un-comment them.